core/src/main/java/com/linecorp/armeria/server/HttpServerHandler.java

```diff
@@ -91,6 +92,7 @@
     private static final String MSG_INVALID_REQUEST_PATH = HttpStatus.BAD_REQUEST + "\nInvalid request path";
 
     private static final HttpData DATA_INVALID_REQUEST_PATH = HttpData.ofUtf8(MSG_INVALID_REQUEST_PATH);
+    private final Consumer<ServerConfig> configUpdateListener = this::swapServerConfig;
```

Comment:
Could you group this field with other member fields rather than static ones?

Reply:
Yes.